english/swami-vivekananda.md

```diff
@@ -0,0 +1,5 @@
+﻿Want to know some of Swami Vivekananda's teachings? [Refer this](./teachings-of-swami-vivekananda/quotes.md)

```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.